unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Common/ShowHideAnimator/ShowHideAnimator.cs

```diff
@@ -1,21 +1,20 @@
-using System;
+using JetBrains.Annotations;
```

Comment:
I'm not sure if we should be adding this kind of `JetBrains` dependencies to an open-source project... I think we should discuss it with the team.

Reply:
as discussed, researched, and answered from JetBrains - it is also OpenSource and under MIT License. [Here ](https://rider-support.jetbrains.com/hc/en-us/community/posts/7665176733458-Is-it-allowed-to-use-JetBrains-Annotations-in-the-OpenSource-project-) is they reply to my question in the forum 
for team discussion with Alex, Kinerius, Nicco - follow this [link to slack thread](https://decentralandteam.slack.com/archives/C03G496L6NN/p1663083874535079)